### ISSUE TITLE
Add Code Of Conduct and Contibuting.md

### ISSUE THREAD

**Issue by Yuheshpandian** [kyclark/tiny_python_projects#47]
Hi @Kyclark I think your repository is fabulous but its missing the following files,

- CODE_OF_CONDUCT.md
- Contirbuting.md

I think these are essential to your repository as you expect contributions

Correct if i'm wrong or if any reasons why you didn't add those files

Thank you

**Comment by Aks11-11** [comment]
agree

**Comment by Yuheshpandian** [comment]
Yeah I too think these 2 files are very for a good professional Repository... Also if you want I can Contribute regarding it

**Comment by Vruddhi18** [comment]
I agree with this comment! The addition of a CODE_OF_CONDUCT.md and CONTRIBUTING.md would definitely enhance the repository, especially for potential contributors.
